addon/components/frost-select-li.js

```diff
@@ -22,10 +35,16 @@ export default Component.extend(PropTypeMixin, {
     onSelect: PropTypes.func.isRequired
   },
 
+  /** @returns {Object} the default property values when not provided by consumer */
```

Comment:
b/c I was making the components look as they would when the generator runs. Again, for the same reason as the empty comment blocks, when changes are made, I think it makes sense to have the proper place to put things already there.